.github/workflows/tests.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           node-version: 14.x
       - uses: browser-actions/setup-chrome@latest
         with:
-          chrome-version: 982053
```

Comment:
I wonder if this will break when the Chrome version exceeds the Chromedriver version? I think the dependencies are installed exactly according to the `package-lock.json` on CI.

Not the end of the world, perhaps we should add a step to run `npm install chromedriver@latest` on CI if we're using `stable`.

Reply:
reverted